frontend/src/components/professor/ChapterDetails/ChapterDetails.js

```diff
@@ -19,12 +19,12 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faArrowDown, faArrowUp, faPenToSquare, faTrash } from '@fortawesome/free-solid-svg-icons'
 import ChapterMap from '../../student/GameMapPage/Map/ChapterMap'
 import DeletionModal from './DeletionModal'
-import EditChapterModal from './EditChapterModal'
 import { getConfigJson } from '../GameManagement/GameLoader/mockData'
 import ChapterService from '../../../services/chapter.service'
 import EditActivityModal from './EditActivityModal'
 import AddActivityModal from './AddActivityModal'
 import { TeacherRoutes } from '../../../routes/PageRoutes'
+import { AddOrEditChapterModal } from '../GameManagement/AddChapterModal/AddOrEditChapterModal'
```

Comment:
W sumie może dajmy nazwę po prostu `ChapterModal` ? Ta nazwa `AddOrEditChapterModal` wydaje mi się głupio brzmieć. Jeżeli zmienisz na `ChapterModal` to zmień na taką samą nazwę folder, w którym jest ten plik. Jeżeli zostawisz jaka jest to zmień nazwę folderu na `AddOrEditChapterModal`